### PR TITLE
Moved postgresql job from jenkins to drone

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
@DeepDiver1975 do we need also to test with the previous version of postgres? (9.6)

### CODE REVIEW

**Comment by SergioBertolinSG** [comment]
Drone error

```
SQLSTATE[42703]: Undefined column: 7 ERROR:  column "min_value" does not exist
LINE 1: SELECT min_value, increment_by FROM "oc_storages_numeric_id_...
```

Which is the expected from https://github.com/owncloud/core/issues/29342

File: .drone.yml [hunk 1]
```diff
@@ -138,7 +138,7 @@ services:
         DB_TYPE: mysql
 
   postgres:
-    image: postgres:9.6
+    image: postgres:10.0
```
Comment (DeepDiver1975) [inline .drone.yml]: Jenkins is testing against 9.4 - please use that version.
especially since we know that postgres 10 is having trouble

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

**Comment by SergioBertolinSG** [comment]
@DeepDiver1975 ready to go?

**Review by DeepDiver1975 — APPROVED** [review]

**Comment by phil-davis** [comment]
Backport stable10 #29517

**Pull request merged**